packages/ui/src/lib/layouts/LayoutEditor.svelte

```diff
@@ -0,0 +1,312 @@
+<script lang="ts">
+import { faBars, faCheck, faGripVertical, faUndo } from '@fortawesome/free-solid-svg-icons';
+import { scale, SvelteMap } from 'svelte/reactivity';
+
+import { Icon } from '../icons';
+import type { LayoutEditItem } from './types';
+
+interface Props {
+  items: LayoutEditItem[];
+  ordering?: Map<string, number>;
```

Comment:
suggestion(non-blocking): you may add a small comment to explain how it can be used